tests/test_config.py

```diff
@@ -15,11 +16,23 @@
 
 
 def test_get_user_config_dir():
-    assert get_user_config_dir() == Path(__file__).parent / "data/user_dirs/config/openeo-python-client"
+    # For Linux/Mac we configure the config and data directories in conftest.py
+    # via the environment variables XDG_CONFIG_HOME and XDG_DATA_HOME.
+    # But on Windows we need to set APPDATA instead, and we don't have separate
+    # variables for config and data.
+    if platform.system() == "Windows":
+        os.environ["APPDATA"] = str(Path(__file__).parent / Path("data/user_dirs/config"))
```

Comment:
ca'tn we just put that `os.environ`  in conftest.py like https://github.com/Open-EO/openeo-python-client/blob/01f525615fead0db67810fc2f1b92f4e15b7a089/tests/conftest.py#L11-L13  ? does that work? even without an "if windows" check?

Reply:
The problem here is that the decision branch for Windows does not use those variables at all. 
Not unless we want to change that.

But more so, Windows does not have separate environment variables for config and data. It has just one: APPDATA.
So for the test you kind of have to set APPDATA to different values in each of the two tests, or alternatively you have to change the test even more to compare with a different expected path.

https://github.com/Open-EO/openeo-python-client/blob/master/openeo/config.py#L22-L42

Reply:
ok I understand now.

I would then still set (for example):
```python
os.environ["APPDATA"] = str(Path(__file__).parent / "\\AppData\\Roaming"
```
in conftest,py (so that this config treats different os'es more equally)

and change the expected values in the tests accordingly, e.g.

```python
if platform.system() == "Windows":
    expected =...
else:
    expected = ...
assert get_user_config_dir() == expected
```

Reply:
Then the expected values in the test also more reflect how it works in reality (that there is no distinction between data and config on windows)

Reply:
Indeed that is cleaner.
Done.